lib/IDE/CompletionInstance.cpp

```diff
@@ -709,9 +709,8 @@ void swift::ide::CompletionInstance::codeComplete(
           CancellationFlag->load(std::memory_order_relaxed)) {
         Callback(ResultType::cancelled());
       } else {
-        MutableArrayRef<CodeCompletionResult *> Results = context.takeResults();
         assert(SwiftContext.swiftASTContext);
-        Callback(ResultType::success({Results, SwiftContext}));
+        Callback(ResultType::success({context.getResultSink(), SwiftContext}));
```

Comment:
This copies `CodeCompletionResultSink` object which has `std::vector<CodeCompletionResult *> Results` etc. Copying it sounds heavy. Do we really want that? If so could you explain why?

Reply:
Good catch `CodeCompleteResult` should store `CodeCompletionResultSink` by reference. Updated it.